ecosystem/node/run-validator.mdx

```diff
@@ -0,0 +1,670 @@
+---
+title: "Run a validator"
+description: "Run a validator node with MyTonCtrl"
+---
+
+import { Aside } from '/snippets/aside.jsx';
+import { Image } from '/snippets/image.jsx';
+
+## Overview
+
+This guide explains how to run a validator TON node with MyTonCtrl from scratch.
+
+<Image
+  src="/resources/images/nodes/run-validator/run-validator-30.png"
+  darkSrc="/resources/images/nodes/run-validator/run-validator-30.png"
+  alt="Architecture"
+/>
+
+## Step 1: Prepare environment
+
+### 1.1 Maintain costs and expenses
+
+- 200 TON per month on the validator hot wallet for its operational transactions.
+- Validator deposit stake [700 000 TON \~ 4 000 000 TON](/ecosystem/node/run-validator#step-4:-set-optimal-stake-for-validator).
+- 100 TB/month traffic at a peak load.
+
+### 1.2 Minimal hardware requirements
+
+- 16-core CPU
+- 128 GB RAM
+- 1TB NVMe SSD or provisioned 64+k IOPS storage
+- 1 Gbit/s network connectivity
+- Public IP address (fixed IP address)
+
+<Aside type="tip">
+  A 1 Gbit/s network connection is recommended to ensure reliable performance during peak load conditions. The average expected bandwidth usage is approximately 100 Mbit/s.
+  Validators must carefully consider IOPS disk requirements, as insufficient storage performance can severely impact validator stability and overall network health.
+</Aside>
+
+### 1.3 Software requirements
+
+- Ubuntu 22.04 LTS or 24.04 LTS
+- Python 3.10
+- Clang 16.0.0
+
+<Image
```

Comment:
We should write such things as a text, as it is easy to read (and text is the only source of information for search engine and mintlify AI chat). Split into three code blocks if necessary: "check ubuntu version", "check python version", "check clang version".